app/src/main/res/drawable/ic_launcher_background.xml

```diff
@@ -1,74 +1,170 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
why do you change the launcher icon?  Have you used it somewhere?

Seems you found a way to format whole project. And that is nice, but should be done in a separate pull request. Because what now happens - I need to check every line in every file you changed

It still has a reason why I shared that marco